coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Product/UpdateProductRequest.cs

```diff
@@ -0,0 +1,29 @@
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace CoffeeCard.Models.DataTransferObjects.v2.Product
+{
+    /// <summary>
+    /// Initiate a new product request
+    /// </summary>
+    public class UpdateProductRequest
```

Comment:
The DataAnnotations I have suggested for AddProduct should be applied on update as weel